aiida/cmdline/groups/dynamic.py

```diff
@@ -43,20 +44,22 @@ def cmd_create():
 
     def __init__(
         self,
-        command,
+        command: t.Callable,
         entry_point_group: str,
         entry_point_name_filter: str = r'.*',
         shared_options: list[click.Option] | None = None,
         **kwargs
     ):
         super().__init__(**kwargs)
-        self.command = command
+        # NOTE: We should probably fix this properly, mypy complains:
+        # "error: Cannot assign to a method"
+        self.command = command  # type: ignore[assignment]
```

Comment:
I think this is because the parent class of `VerdiCommandGroup` is `click.Group` which has a `command` method, which we are overriding here with a variable. I think changing this to
```suggestion
        self._command = command
```
should work. And then update elsewhere in the code where `.command` is used of course.

Reply:
That did the trick, thanks! I wasn't sure if this is safe i.e. if we're not relying on the fact that we're overriding this method but looking at the code here and the parent class in click it seems fine.

Reply:
I think `mypy` actually did its job this case and showed us that we were (unintentionally) overriding a method